recipes/python-ninja/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "python-ninja" %}
+{% set version = "1.10.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/n/ninja/ninja-{{ version }}.tar.gz
+  sha256: e1b86ad50d4e681a7dbdff05fc23bb52cb773edb90bc428efba33fa027738408
+
+build:
+  noarch: python
```

Comment:
Cannot be `noarch: python` if compiled extensions, like the `ninja` executable are contained.

Reply:
Thanks @awvwgk, Ci tests all pass now :smiley: